pkgs/development/libraries/harfbuzz/default.nix

```diff
@@ -1,9 +1,11 @@
 { stdenv, fetchurl, pkgconfig, glib, freetype, cairo, libintl
 , icu, graphite2, harfbuzz # The icu variant uses and propagates the non-icu one.
+, gobject-introspection ? null
 , ApplicationServices, CoreText
 , withCoreText ? false
 , withIcu ? false # recommended by upstream as default, but most don't needed and it's big
 , withGraphite2 ? true # it is small and major distros do include it
+, withGObject ? true # needed by pango (~0.2MiB closure size increase)
```

Comment:
Is there a reason someone might need to disable it? I would prefer just including it unconditionally if there are no obstacles.

Reply:
No particular reason, I did it because all other options are configurable.

Reply:
Done.